File: setima/python/3.py
  matriz_A = [[int(input(f"Digite o elemento da posição ({i + 1}, {j + 1}): ")) for j in range(15)] for i in range(15)]

soma_pares = sum(matriz_A[i][i] for i in range(15) if matriz_A[i][i] % 2 == 0)
print(f"A soma dos elementos pares na diagonal principal é: {soma_pares}")

File: setima/c/3.c
#include <stdio.h>

int main() {
    int matriz_A[15][15];
    int soma_pares = 0;

    for (int i = 0; i < 15; i++) {
        for (int j = 0; j < 15; j++) {
            printf("Digite o elemento da posição (%d, %d): ", i + 1, j + 1);
            scanf("%d", &matriz_A[i][j]);

            if (i == j && matriz_A[i][j] % 2 == 0) {
                soma_pares += matriz_A[i][j];
            }
        }
    }

    printf("A soma dos elementos pares na diagonal principal é: %d\n", soma_pares);

    return 0;
}
